components/scream/src/physics/p3/p3_iso_c.f90

```diff
@@ -143,6 +139,10 @@ subroutine p3_main_c(qc,nc,qr,nr,th_atm,qv,dt,qi,qm,ni,bm,   &
 
     real(kind=c_real), dimension(its:ite,kts:kte,49)   :: p3_tend_out
     real(kind=c_real), dimension(its:ite,3) :: col_location
+    real(kind=c_real), dimension(its:ite,kts:kte)      :: mu_c, lamc
+    real(kind=c_real), dimension(its:ite,kts:kte)      :: precip_total_tend
+    real(kind=c_real), dimension(its:ite,kts:kte)      :: nevapr
```

Comment:
Isn't nevapr supposed to be deleted everywhere?

Reply:
oh yes, I think this PR was opened before the cleanup PR went in.  I can clean that up

Reply:
it isn't deleted in the F90 code, right?  At least not in this PR.  I can go ahead and make that change too if you would like.